### DISCUSSION TITLE
Turn off screensaver, screen lock and display shutoff in the INSTALLER

### DISCUSSION THREAD

**Discussion by jtcraigjr** [community/community#77821] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have seen a lot of new Linux Mint users fresh from Windows, get stuck with an Install that Stops Dead.
Would it be possible for the Installer to be coded so that it would Automatically Turn off screensaver, screen lock and display shutoff, and Power Manager?
thank you.; John

**Comment by dakshithadissanayaka** [comment]
Disabling screensaver, screen lock, display shutoff, and power manager during the installation process can be a helpful improvement for new Linux Mint users who are transitioning from Windows. This would prevent the installation from being interrupted due to these settings and ensure a smoother installation experience.

Here's a possible approach to implementing this feature in the installer:

1. **Detect Screensaver Status:** Upon starting the installation process, detect whether the screensaver is active. If it is, disable the screensaver using the appropriate system commands or configuration tools.

2. **Disable Screen Lock:** Check if the screen lock is enabled. If it is, disable the screen lock temporarily using the appropriate system commands or configuration tools.

3. **Prevent Display Shutoff:** Disable any settings that could cause the display to shut off during the installation, such as power management settings or idle timeouts. This can be done by setting the display timeout to a high value or disabling power management entirely.

4. **Handle Power Manager:** If necessary, adjust power management settings to prevent the system from entering sleep or hibernation mode during the installation. This may involve modifying power management profiles or changing specific settings.

5. **Revert Changes Post-Installation:** Once the installation is complete, revert any changes made to screensaver, screen lock, display shutoff, and power manager settings to their original state. This ensures that the user's preferred settings are not permanently altered.

Implementing these measures would significantly enhance the installation experience for new Linux Mint users by preventing disruptions due to screensaver, screen lock, display shutoff, or power management settings.

**Comment by jtcraigjr** [comment]
a NEW user, from WINDOWS, is using a mint.iso and all of those features are turned ON by default in the Installer.
there will NOT be any user profiles carried over from a prior installation.

Your process makes a LOT of sense, but... a new user from Windows is not aware of this stuff.

This has irked me more than I care to complain about.....
when the screen goes blank, I hit the Power OFF key, and start all over, and piss and moan that I forgot to turn those features OFF

and this is NOT my first rodeo installing Linux ISOs....
it is a Major Fault with the Installer Developers for not considering the New Users, or those of us who have to install an ISO many times a day.

**Reply by dakshithadissanayaka** [reply]
I understand your frustration with the default settings in the Linux Mint ISO installer. It's understandable that a new user from Windows might not be familiar with these settings and might be surprised to find them enabled by default. This can lead to confusion and frustration, especially if the user is not aware of the options available to them.

I agree that the installer developers could make some improvements to make the process more user-friendly for newcomers. For example, they could provide more clear and concise explanations of the default settings and give users the option to disable them easily. Additionally, they could provide a more streamlined installation process that doesn't require users to make as many decisions.

Here are a few suggestions for how to improve the installer experience for new users:

* **Provide clear and concise explanations of the default settings:** The installer should provide clear and concise explanations of each default setting so that users understand what they are enabling or disabling. This could be done with tooltips, pop-ups, or even a dedicated help screen.
* **Give users the option to disable default settings easily:** Users should be able to easily disable any default settings that they do not want. This could be done by providing a simple checkbox or toggle switch for each setting.
* **Provide a more streamlined installation process:** The installer could be streamlined to reduce the number of decisions that users need to make. This could be done by pre-selecting some of the default settings or by providing a guided installation process.

Overall, I think it's important for the Linux Mint developers to consider the needs of new users when designing the installer experience. By making the process more user-friendly, they can help to reduce the frustration that some users experience and make Linux Mint more accessible to a wider audience.

**Comment by queenofcorgis** [comment]
Hi @jtcraigjr ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Linux Mint. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
